add SR2019 Southampton kickstart risk assessment

Adimote opened the pull request.

Adimote committed:
add SR2019 Southampton kickstart risk assessment

Adimote committed:
add fibreglass risk

Adimote committed:
fix the fibreglass compile error

PeterJCLaw commented on template.tex:
```diff
@@ -0,0 +1,211 @@
+likelihood rating of $3$, an impact rating of $1$, and an overall risk level of
+$3 \times 1 = 3$.
+
+These guidelines are based on those provided in Union Southampton's risk
```
What is "Union Southampton"?

Adimote committed:
Remove Union Southampton

Adimote committed:
Update template.tex

Rename Union Southampton to 'University of Southampton
Student Union', as it keeps changing name.

Adimote commented:
out of time, merging this now
